internal/sync/secrets_sync.go

```diff
@@ -123,5 +123,12 @@ func SyncDestinationDelete(ctx context.Context, d *schema.ResourceData, meta int
 }
 
 func secretsSyncDestinationPath(name, typ string) string {
-	return fmt.Sprintf("sys/sync/destinations/%s/%s", typ, name)
+	var path string
+	switch typ {
+	case "github-apps":
+		path = fmt.Sprintf("sys/sync/%s/%s", typ, name)
```

Comment:
Since this isn't a destination path, I'd suggest having a dedicated method (`secretsSyncGitHubAppsPath`?) to avoid confusion.